metric/blockgroup/dCPMTS.csv

```diff
@@ -1,154 +1,154 @@
-370630020212,0.414428532,0.414428532,0.414428532,0.414428532,0.414428532,0.414428532,0.414428532,0.414428532
-370630020221,0.476114607,0.476114607,0.476114607,0.476114607,0.476114607,0.476114607,0.476114607,0.476114607
-370630020222,0.527096428,0.527096428,0.527096428,0.527096428,0.527096428,0.527096428,0.527096428,0.527096428
-370630020231,0.357921069,0.357921069,0.357921069,0.357921069,0.357921069,0.357921069,0.357921069,0.357921069
-370630020232,0.531750112,0.531750112,0.531750112,0.531750112,0.531750112,0.531750112,0.531750112,0.531750112
-370630020241,1.185931986,1.185931986,1.185931986,1.185931986,1.185931986,1.185931986,1.185931986,1.185931986
-370630020242,0.226600741,0.226600741,0.226600741,0.226600741,0.226600741,0.226600741,0.226600741,0.226600741
-370630020243,0.266859576,0.266859576,0.266859576,0.266859576,0.266859576,0.266859576,0.266859576,0.266859576
-370630020251,0.856800629,0.856800629,0.856800629,0.856800629,0.856800629,0.856800629,0.856800629,0.856800629
-370630020252,0.875012186,0.875012186,0.875012186,0.875012186,0.875012186,0.875012186,0.875012186,0.875012186
-370630020253,0.64207343,0.64207343,0.64207343,0.64207343,0.64207343,0.64207343,0.64207343,0.64207343
-370630020261,0.594458974,0.594458974,0.594458974,0.594458974,0.594458974,0.594458974,0.594458974,0.594458974
-370630020262,1.634090705,1.634090705,1.634090705,1.634090705,1.634090705,1.634090705,1.634090705,1.634090705
-370630020271,2.73599029,2.73599029,2.73599029,2.73599029,2.73599029,2.73599029,2.73599029,2.73599029
-370630020272,3.394428744,3.394428744,3.394428744,3.394428744,3.394428744,3.394428744,3.394428744,3.394428744
-370630020273,0.294960583,0.294960583,0.294960583,0.294960583,0.294960583,0.294960583,0.294960583,0.294960583
-370630020281,4.345266505,4.345266505,4.345266505,4.345266505,4.345266505,4.345266505,4.345266505,4.345266505
-370630020282,0.268706107,0.268706107,0.268706107,0.268706107,0.268706107,0.268706107,0.268706107,0.268706107
-370630020283,3.748872155,3.748872155,3.748872155,3.748872153,3.748872153,3.748872153,3.748872153,3.748872153
-370630021001,18.24984792,18.24984792,18.24984792,18.24984792,18.24984792,18.24984792,18.24984792,18.24984792
-370630021002,33.73147697,33.73147697,33.73147697,33.73147697,33.73147697,33.73147697,33.73147697,33.73147697
-370630021003,19.46911853,19.46911853,19.46911853,19.46911853,19.46911853,19.46911853,19.46911853,19.46911853
-370630022001,0.593992964,0.593992964,0.593992964,0.593992964,0.593992964,0.593992964,0.593992964,0.593992964
-370630023001,0.250494356,0.250494356,0.250494356,0.250494356,0.250494356,0.250494356,0.250494356,0.250494356
-370630023002,0.223979257,0.223979257,0.223979257,0.223979257,0.223979257,0.223979257,0.223979257,0.223979257
-370639801001,8.233348864,8.233348864,8.233348864,8.233348866,8.233348866,8.233348866,8.233348866,8.233348866
+id,y_2012,y_2013,y_2014,y_2015,y_2016,y_2017,y_2018,y_2019,y_2020
```

Comment:
Why was all of the data for dCPMTS replaced?

Reply:
these are tract area in square miles, and I switched to rounding them to 5 decimal places rather than reporting out to 9 decimal places. I don't think that should impact the compass values since there's still 6 significant digits in the denominator, but honestly I haven't thought about significant digits since middle school so lmk if that makes sense in your opinion too?